app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/AnnotationWithCalloutActivity.kt

```diff
@@ -112,11 +113,15 @@ class AnnotationWithCalloutActivity : AppCompatActivity() {
         viewAnnotationManager.updateViewAnnotation(
           viewAnnotation,
           viewAnnotationOptions {
-            width(viewAnnotationManager.getViewAnnotationOptionsByView(viewAnnotation)?.width!! + pxDelta)
-            height(viewAnnotationManager.getViewAnnotationOptionsByView(viewAnnotation)?.height!! + pxDelta)
             selected(isSelected)
           }
         )
+        (button.layoutParams as ViewGroup.MarginLayoutParams).apply {
+          bottomMargin += pxDelta
```

Comment:
What's about top margin?

Reply:
yeah, perhaps adding some value to all the margins will read better and look the same, will try

Reply:
will leave as is for now, it does not actually make that much diff if updating all the margins